### PR TITLE
Assignment 4 completed

### PR DESCRIPTION
**Pull request by santi-gv**

### CODE REVIEW

File: Assignment4/santi-gv-190031/task06.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/github/FacultadInformatica-LinkedData/Curso2022-2023/blob/master/Assignment4/course_materials/notebooks/Task06.ipynb
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib
```
Comment (dgarijo) [inline Assignment4/santi-gv-190031/task06.py]: this won't work in python. It's notebook magic

File: Assignment4/santi-gv-190031/task06.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+# Visualize the results
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+# TO DO
+johnUri = ns.JohnSmith
+g.add((ns.UPM, RDF.type, ns.University))
+g.add((johnUri, VCARD.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/santi-gv-190031/task06.py]: vcard does not define "work" as a property. Define your own

**Review by dgarijo — CHANGES_REQUESTED** [review]
Very small change requested.

**Comment by santi-gv** [comment]
Errors fixed

File: Assignment4/santi-gv-190031/task06.py (hunk 2 above)
Comment (santi-gv) [reply Assignment4/santi-gv-190031/task06.py]: resolved

**Pull request merged**